srcpkgs/hydrus/files/hydrus-client

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec python -OO /usr/lib/hydrus/client.pyw "$@"
```

Comment:
python is python2, this should use python3. Doesn't a simple symlink also work instead of this wrapper?

Reply:
`/usr/bin/hydrus -> /usr/lib/hydrus/client.pyw` ?
yea, could work. ill give it a shot and change if it works.

Reply:
symlinks work just fine
client and server.py use python3 shebang